### PR TITLE
Adding AE to items Round 1

### PR DESCRIPTION
**Pull request by hooksta4**
**_I affirm:_**
- [x] I have paid attention to this example and will edit again if need be to not break the formatting, or I will be ignored
- [x] that I agree to LandSandBoat's [Limited Contributor License Agreement](https://github.com/LandSandBoat/server/blob/base/.github/CONTRIBUTOR_AGREEMENT.md), as written on this date
- [x] that I have **read the [Contributing Guide](https://github.com/LandSandBoat/server/blob/base/CONTRIBUTING.md)**
- [x] that I've _**tested my code and things my code changed**_ since the last commit in the PR, and will test after any later commits

### CODE REVIEW

**Comment by ghost** [comment]
I will say that need to fix some of the Styles

Example 

Don't 
--INSERT INTO `item_mods` VALUES (17774,431,1);   -- Additional effect: Light Damage (Type DAMAGE)--shiranui 17774

Do
-- INSERT INTO `item_mods` VALUES (17774,431,1);   -- Additional effect: Light Damage (Type DAMAGE) -- shiranui 17774

Where you have -- there needs to be space you can find the error that show up in Style Check for ones to fix

At least that what I see currently

**Comment by hooksta4** [comment]
> 17774,431,1

JFC, the one thing I specifically went in to fix I didnt smh Fix incoming. My eyes are bleeding.

File: sql/item_mods.sql [hunk 1]
```diff
@@ -20687,16 +20687,16 @@ INSERT INTO `item_mods` VALUES (16934,25,5);
...
-INSERT INTO `item_mods` VALUES (16936,431,1);   -- Additional effect: Darkness Damage (Type DAMAGE)--demonic_sword 16936
+INSERT INTO `item_mods` VALUES (16936,431,1);   -- Demonic Sword (Additional effect: Darkness Damage)
 INSERT INTO `item_mods` VALUES (16936,499,8);   -- Additional effect animation (subEffect)
-INSERT INTO `item_mods` VALUES (16936,500,21);   -- Additional effect damage
+INSERT INTO `item_mods` VALUES (16936,500,21);  -- Additional effect damage
 INSERT INTO `item_mods` VALUES (16936,501,10);  -- Additional effect Chance 10%
 INSERT INTO `item_mods` VALUES (16936,950,8);   -- Additional effect element Darkness (xi.magic.element)
 INSERT INTO `item_mods` VALUES (16937,16,8); -- ICE_RES
-INSERT INTO `item_mods` VALUES (16937,431,1);   -- Additional effect: Ice Damage (Type DAMAGE)--icicle_sword 16937
+INSERT INTO `item_mods` VALUES (16937,431,1);   --Icicle Sword (Additional effect: Ice Damage)
```
Comment (RuthlessMortal) [inline sql/item_mods.sql]: Need space on  --Icicle Sword

**Comment by hooksta4** [comment]
Yea just fixed it too. I’m going to bed been staring at it too long today.

File: sql/item_mods.sql [hunk 2]
```diff
@@ -20442,28 +20622,82 @@ INSERT INTO `item_mods` VALUES (16894,9,3);
...
 INSERT INTO `item_mods` VALUES (16936,22,8); -- DARK_RES
-INSERT INTO `item_mods` VALUES (16936,431,1);
+INSERT INTO `item_mods` VALUES (16936,431,1);   -- Demonic Sword (Additional effect: Darkness Damage)
+INSERT INTO `item_mods` VALUES (16936,499,8);   -- Additional effect animation (subEffect)
+INSERT INTO `item_mods` VALUES (16936,500,21);  -- Additional effect damage
+INSERT INTO `item_mods` VALUES (16936,501,10);  -- Additional effect Chance 10%
+INSERT INTO `item_mods` VALUES (16936,950,8);   -- Additional effect element Darkness (xi.magic.element)
 INSERT INTO `item_mods` VALUES (16937,16,8); -- ICE_RES
-INSERT INTO `item_mods` VALUES (16937,431,1);
+INSERT INTO `item_mods` VALUES (16937,431,1);   --Icicle Sword (Additional effect: Ice Damage)
```
Comment (Xaver-DaRed) [inline sql/item_mods.sql:20696]: Missed one.

File: sql/item_mods.sql [hunk 3]
```diff
@@ -21875,7 +22223,7 @@ INSERT INTO `item_mods` VALUES (18008,501,10);  -- Additional effect Chance 10%
 INSERT INTO `item_mods` VALUES (18008,951,6);   -- Additional effect Status ID xi.effect.SILENCE
 INSERT INTO `item_mods` VALUES (18008,952,5);   -- Additional effect Status Potency 5
 INSERT INTO `item_mods` VALUES (18008,953,30);  -- Additional effect Status Duration 30sec
-INSERT INTO `item_mods` VALUES (18009,345,1000);  -- Martial Knife: TP Bonus +1000
+INSERT INTO `item_mods` VALUES (18009,345,1000);-- Martial Knife: TP Bonus +1000
```
Comment (Xaver-DaRed) [inline sql/item_mods.sql:22226]: Missed another here.

File: sql/item_mods.sql [hunk 4]
```diff
@@ -22182,8 +22605,12 @@ INSERT INTO `item_mods` VALUES (18211,11,-2);
...
 INSERT INTO `item_mods` VALUES (18213,10,1);
 INSERT INTO `item_mods` VALUES (18213,11,1);
-INSERT INTO `item_mods` VALUES (18220,431,1);
-INSERT INTO `item_mods` VALUES (18221,345,1000);  -- Martial Bhuj: TP Bonus +1000
+INSERT INTO `item_mods` VALUES (18220,431,1);   -- Prominence Axe ((Additional effect: Fire Damage))
+INSERT INTO `item_mods` VALUES (18220,499,1);   -- Additional effect animation (subEffect)
+INSERT INTO `item_mods` VALUES (18220,500,10);  -- Additional effect damage
+INSERT INTO `item_mods` VALUES (18220,501,5);   -- Additional effect Chance 10%
+INSERT INTO `item_mods` VALUES (18220,950,1);   -- Additional effect element Fire (xi.magic.element)
+INSERT INTO `item_mods` VALUES (18221,345,1000);-- Martial Bhuj: TP Bonus +1000
```
Comment (Xaver-DaRed) [inline sql/item_mods.sql:22613]: And another one here.

**Review by Xaver-DaRed — CHANGES_REQUESTED** [review]
The changes themselves seem reasonable. Just fix the 3 missing spaces and I'd say this is good to go.

File: sql/item_mods.sql [hunk 5]
```diff
@@ -22282,23 +22714,33 @@ INSERT INTO `item_mods` VALUES (18287,356,73); -- Onslaught
...
+INSERT INTO `item_mods` VALUES (18289,431,1);  -- Additional Effect
 INSERT INTO `item_mods` VALUES (18289,506,50); -- Extra DMG Chance 5%
 INSERT INTO `item_mods` VALUES (18289,507,250); -- Extra DMG X2.5
 INSERT INTO `item_mods` VALUES (18293,256,6); -- Stage 4 Bravura - Aftermath
 INSERT INTO `item_mods` VALUES (18293,356,89); -- Metatron Torment
 INSERT INTO `item_mods` VALUES (18294,25,20); -- Bravura LV75 - ACC +20
 INSERT INTO `item_mods` VALUES (18294,256,6); -- Aftermath
 INSERT INTO `item_mods` VALUES (18294,355,89); -- Metatron Torment
-INSERT INTO `item_mods` VALUES (18294,431,1); -- Additional Effect
+INSERT INTO `item_mods` VALUES (18294,431,2);   -- BROvura (Additional effect: EVASION_DOWN)
```
Comment (tristamgreen) [inline sql/item_mods.sql:22736]: Brovura

Comment (hooksta4) [reply sql/item_mods.sql:22736]: weird typo - will fix it in next round.

Comment (hooksta4) [reply sql/item_mods.sql:22736]: actually, upon consideration, i will not change it as its a comment and someone else will have to PR an A instead of an O

Comment (TeoTwawki) [reply sql/item_mods.sql:22736]: image

:joy:

Comment (TeoTwawki) [reply sql/item_mods.sql:22736]: the great axe that calls everyone dude or bro: dudura

Comment (hooksta4) [reply sql/item_mods.sql:22736]: should have made it bruhvura then

Comment (TeoTwawki) [reply sql/item_mods.sql:22736]: image

Comment (TeoTwawki) [reply sql/item_mods.sql:22736]: i ninja'd a commit to your branch

**Review by Xaver-DaRed — APPROVED** [review]

**Pull request merged**